integration/spark-common-test/src/test/scala/org/apache/carbondata/integration/spark/testsuite/dataload/TestLoadDataGeneral.scala

```diff
@@ -275,6 +279,34 @@ class TestLoadDataGeneral extends QueryTest with BeforeAndAfterEach {
       CarbonCommonConstants.BLOCKLET_SIZE_DEFAULT_VAL)
   }
 
+  test("block concurrent load if DICTIONARY_INCLUDE is specified") {
```

Comment:
Though it happens in most cases, This test cannot ensure parallel data load case. I think better to remove testcase.